concepts/40 Angular Components/10 Getting Started/03 Add DevExtreme to an Angular CLI Application/08 Register 3rd-Party Dependencies in Angular CLI 6+/01 JSZip Registration.md

```diff
@@ -1,4 +1,4 @@
-Register the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a> in the `tsconfig.json` file:
+DevExtreme requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip library</a>. Since JSZip v3.3.0, the library does not need registration. If you use an earlier version, register JSZip in the `tsconfig.json` file:
```

Comment:
the library does not need registration?
maybe, the library does not need to be registered OR you don't need to register the library
this note applies to the same sentence below